### ISSUE TITLE
Stops displaying messages upon ngrok tunnel expires 

### ISSUE THREAD

**Issue by artemious7** [Microsoft/BotFramework-Emulator#1194]
If I don't use the Bot Framework Emulator for a few hours, but it stays open, ngrok tunnel expires when I come back and I see a notification about this. 

The chat window is still **able to send messages** to my localhost bot and **even to receive replies**, but the replies are **not displayed in the chat** window. I can see them only in the LOG window, and no errors. 

I have to either restart the conversation with same user identity, or restart the app. Reconnecting to the
 ngrok tunnel doesn't help.

**Comment by justinwilaby** [comment]
ngrok is bypassed with localhost connections.

@artemious7 - Do you have multiple chat windows open with at least one connecting to a remote bot? Does the "reconnect" button in the notifications panel not work?

**Comment by artemious7** [comment]
You are right, I forgot it's bypassed, but anyway it is somehow related. Or maybe there is some other time out...

I remember having only one chat window. 
Reconnect button works. But I didn't have to do that on v3.
